Tasks/SonarQubePostTest/SummaryReport/ReportBuilder.ps1

```diff
@@ -95,45 +90,12 @@ function CreateReport
     return $reportPath
 }
 
-function GetQualityGateVisualDetails
+function FireUploadReportCommand
 {
-    param([string]$qualityGateStatus, [ref][string]$color, [ref][string]$label)
-    
-    
-    switch ($qualityGateStatus) {
-        {$_ -eq "ok"} 
-        { 
-            $color.Value = "#85BB43"
-            $label.Value = "Passed" 
-            break;           
-        }
-        {$_ -eq "warn"} 
-        {
-            $color.Value = "#f90"
-            $label.Value = "Warning" 
-            break;            
-        }
-        {$_ -eq "error"} 
-        {
-            $color.Value = "#d4333f"
-            $label.Value = "Failed"
-            break;  
-        }
-        {$_ -eq "none"} 
-        {
-            $color.Value = "#bbb" 
-            $label.Value = "None"
-            break;  
-        }
-        Default 
-        {
-            Write-Warning "Could not detect the quality gate status or a new status has been introduced."
-            $color.Value = "#bbb" 
-            $label.Value = "Unknown"
-            break;  
-        }
-    }
+    param($reportPath)
     
+    Write-Host "##vso[task.addattachment type=Distributedtask.Core.Summary;name=SonarQube Analysis Report;]$reportPath"
```

Comment:
I guess we're only expecting one analysis report per build definition so attachment name doesn't need to be disambiguated e.g. by the project name.
I wonder what would happen if a user tried including multiple begin-build-end tasks sequence in the same build definition..